### DISCUSSION TITLE
Syncthing Add-on - can't synch backups

### DISCUSSION THREAD

**Discussion by ffgambler** [Poeschl/Hassio-Addons#387] [Q&A]
Hi All,
I hope this is the right place for ma question. I woudl like to synch my backup folder. It looks like Syncthing is not detecting any files in it and no files are shared to another device. I'm I doing something wrong?
![tea53hjpnzka1](https://user-images.githubusercontent.com/122104988/224406460-6c06211a-a5f0-4e2d-a199-5c2b3d210d5c.png)

**Comment by Ionshard** [comment]
I know this is a necro but I ended up here looking for info on how to do this, so figured I should answer it. Inside the Syncthing container the backups folder is mounted at `/backup` not `/root/backup` so make sure you share that folder.
